net/vpnbypass/Makefile

```diff
@@ -5,7 +5,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpnbypass
 PKG_VERSION:=1.3.1
-PKG_RELEASE:=10
+PKG_RELEASE:=9
```

Comment:
you go from 10 to 9?

Reply:
Ooops, my bad, don't know how I missed it.